Tests/TuistGeneratorTests/Generator/SchemeDescriptorsGeneratorTests.swift

```diff
@@ -755,6 +756,7 @@ final class SchemeDescriptorsGeneratorTests: XCTestCase {
         XCTAssertEqual(result.storeKitConfigurationFileReference, .init(identifier: "../nested/configuration/configuration.storekit"))
         XCTAssertEqual(result.locationScenarioReference?.referenceType, "1")
         XCTAssertEqual(result.locationScenarioReference?.identifier, "New York, NY, USA")
+        XCTAssertEqual(result.language, "pl")
```

Comment:
nit: I'd also add `XCTAssertNil(result.language)` to at least one of the other test cases to assert the default is what we expect